pkg/contract_watcher/converter/converter.go

```diff
@@ -90,6 +90,9 @@ func (c *Converter) Convert(logs []gethTypes.Log, event types.Event, headerID in
 				if len(b) == 32 {
 					seenHashes = append(seenHashes, common.HexToHash(strValues[fieldName]))
 				}
+			case uint64:
```

Comment:
Not really related to this PR, but the other thing I noticed about the relevant Oasis events is that they also take arguments of the `ERC20` type. I wonder how we'll want to handle those 🤔

Reply:
Based on a quick test locally, seems like it already parses `ERC20` vals correctly as addresses /shrug